.changes/unreleased/NOTES-20230313-131455.yaml

```diff
@@ -0,0 +1,5 @@
+kind: NOTES
+body: 'all: Acceptance tests have been migrated to [terraform-plugin-testing](https://github.com/hashicorp/terraform-plugin-testing)'
```

Comment:
I think we can avoid changelog entries for testing, unless there's some practitioner benefit/concern. 👍